### DISCUSSION TITLE
VUE ENVIRONMENT VARIABLES

### DISCUSSION THREAD

**Discussion by Behtsa** [chatwoot/chatwoot#6951] [Self hosted]
Hi team, we have the following issue: 

We have our chatwood forked project image created in ECR and set environment variables on AWS ECS container task definition but Environment Variables are not accessible by `process.env`. Do you have any idea on how to solve this issue?

**Comment by pato-1441** [comment]
Hey! Have you found a way to sort this? I'm trying to set some environment variables in Vue too and I found nothing about this
